### DISCUSSION TITLE
Vue.js 3 Transitions

### DISCUSSION THREAD

**Discussion by sistematico** [antfu/vitesse#162] [Q&A]
How to accomplish Vue.js [Router transitions](https://next.router.vuejs.org/guide/advanced/transitions.html) with Vitesse?

**Comment by claytonfaria** [comment]
Hello, you have probably solved your question, however for those arriving here now looking for the same subject. Please note that besides adding the [transitions in the router](https://next.router.vuejs.org/guide/advanced/transitions.html), it is also necessary to add the respective CSS. [Here for more info ](https://v3.vuejs.org/guide/transitions-enterleave.html)

**Reply by ctholho** [reply]
Also, if you are using the default `vite-plugin-vue-layouts` read more about [transitions with layouts](https://github.com/JohnCampionJr/vite-plugin-vue-layouts#transitions)
